docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+
+The minimum base fee that can be used by a transaction. It also shows how much gas will it cost to do native token transfers. The value is in wei and set to `25000000000` in C-Chain. It corresponds to 25 nAvax (gwei). You can use [Snowtrace unitconverter](https://snowtrace.io/unitconverter) to convert between units.
+
+#### `targetGas`
+
+The targeted amount of gas (including block gas cost) to consume within a rolling 10s window.
+
+#### `baseFeeChangeDenominator`
+
+The base fee can change over time. If the parent block used **more** gas than its target, the base fee should **increase**. Otherwise, if the parent block used **less** gas than its target, the base fee should **decrease**.
+
+Here are the calculations:
+
+If the parent block used **more** gas than its target:  
+ `baseFee = max(1, parentBaseFee * (parentGasUsed - parentGasTarget) / parentGasTarget / baseFeeChangeDenominator)`
+
+If the parent block used **less** gas than its target:  
+ `baseFee = max(0, parentBaseFee * (parentGasUsed - parentGasTarget) / parentGasTarget / baseFeeChangeDenominator)`
+
+The `baseFeeChangeDenominator` is set to `36` in C-Chain. This value sets the base fee to increase or decrease by a factor of `1/36` of the parent block's base fee.
+
+Setting this value to a larger value means that the base fee will change more gradually. Setting it to a smaller value will make the base fee change more rapidly.
+
+For further information, see [eip1559](https://eips.ethereum.org/EIPS/eip-1559)
+
+#### `minBlockGasCost`
+
+Minimum gas cost a block should cover. This value is set to `0` in C-Chain.
+
+#### `maxBlockGasCost`
+
+Maximum gas cost a block should cover. This value is set to `1,000,000` in C-Chain.
+
+> `minBlockGasCost` and `maxBlockGasCost` bounds the gas cost of a block.
+
+#### `blockGasCostStep`
+
+This value determines the block gas change rate depending on the [`targetBlockRate`](#targetblockrate). If the parent block is produced at the `targetBlockRate`, the block gas cost will stay the same. If the parent block is produced at a **slower** rate, the block gas cost will **decrease**. If the parent block is produced at a **faster** rate, the block gas cost will **increase**. The amount of change is determined by the following formula:
+
+```
+(blockGasCostStep) * (targetBlockRate - parent block production time)
+```
+
+For example, if the `targetBlockRate` is set to 2 seconds, `blockGasCostStep` is set to `10` and the parent block was produced in 5 seconds. The block gas cost will decrease by 30: `10 * (2 - 5) = -30`
+
+This value is set to `200000` in C-Chain.
+
+
+### Validator Fee Recipient
+This configuration allows validators to specify a fee recipient. 
+
+Use the following configuration to enable validators to receive fees. [You can find more information from the Avalanche docs.](https://docs.avax.network/Subnets/customize-a-Subnet#setting-a-custom-fee-recipient)
+
+```json
+{
+  "config": {
+    "allowFeeRecipients": true
+  }
+}
+```
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Fenced code blocks should be surrounded by blank lines.

> MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines

[More info: MD031](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines)